src/mach_o.rs

```diff
@@ -38,6 +38,8 @@ pub struct MachO {
     /// List of addresses and names of external relocations for the dynamic
     /// linker to resolve.
     pub external_relocations: Vec<(u32, String)>,
+    /// Thread state to initialize with (LC_UNIXTHREAD)
+    pub thread_state: ThreadState,
```

Comment:
If possible I'd prefer not to expose stuff from the mach_object crate outside of this module. Can we extract the bit we are interested in (the entry point address) inside this file instead?

Reply:
That is a good idea. Though, I mentioned in an earlier comment that ThreadState may potentially have some additional important stuff for starting execution. For now, I would be happy with just the address being exposed. Ideally, in the future we would want to employ those extra parameters.